src/Icicle/Avalanche/ToJava.hs

```diff
@@ -280,10 +283,15 @@ primTypeOfPrim p
   dble     M.PrimDoubleExp  = Function "Math.exp"
   dble     M.PrimDoubleSqrt = Function "Math.sqrt"
 
-  cast      M.PrimCastIntOfDouble = Function "(int)"
-  cast      M.PrimCastDoubleOfInt = Function "(double)"
-  cast      M.PrimCastStringOfInt = Function "Integer.toString"
-  cast      M.PrimCastStringOfDouble = Function "Double.toString"
+  toInt    M.PrimToIntFloor      = Function "Math.floor"
+  toInt    M.PrimToIntCeiling    = Function "Math.ceil"
+  toInt    M.PrimToIntRound      = Function "Math.round"
+  toInt    M.PrimToIntTruncate   = Function "Math.trunc"
```

Comment:
This doesn't exist

![Savage](http://ambiata.github.io/meme/savages/savage.png)